micro-ui/web/micro-ui-internals/packages/modules/hrms/CHANGELOG.md

```
@@ -1,3 +1,6 @@
+## [1.8.15]  [25-July-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)